### DISCUSSION TITLE
Is the Caps Lock warning working?

### DISCUSSION THREAD

**Discussion by An-anonymous-coder** [keepassxreboot/keepassxc#11483] [Q&A] [answered]
I didn't want to open an issue for this without first confirming whether or not I'm missing something. [This pull request](https://github.com/keepassxreboot/keepassxc/commit/d9214db404571044ae94f92c9aba0963a0243832) implemented a warning when Caps Lock is enabled. However, this warning does not show up for me. Am I missing anything, or should I make a bug report? Thanks!

**Comment by phoerious** [comment, marked as answer]
Yes, it's working, but there is no Wayland implementation yet. So if you are on Linux with Wayland, you won't have that.
